src/library/basesqltablemodel.cpp

```diff
@@ -515,9 +511,8 @@ void BaseSqlTableModel::setSort(int column, Qt::SortOrder order) {
             QString field = m_tableColumns[column];
             QString sort_field = QString("%1.%2").arg(m_tableName, field);
             m_tableOrderBy.append(sort_field);
-        #ifdef __SQLITE3__
```

Comment:
If we remove __SQLITE3__ here, we should remove this from the whole code base
and re-actor the scons scripts accordingly.